manager/aica_django/converters/Knowledge.py

```diff
@@ -1070,6 +1124,8 @@ def suricata_alert_to_knowledge(
         )
     )
 
+    if alert["alert"]["category"] == "":
```

Comment:
Is this exception caught upstream to avoid the manager aborting?

Reply:
it was used to print out the complete Suricata alert before the information is lost in class KnowledgeNode. I found cases where the exception was raised downstream and have to find out why it crashes, therefor the needed extra information in str(alert)

Reply:
That's fine, just want to make sure it doesn't cause something upstream to break. If this exception is raised here (versus, for example, printing as an error), we'll want to make sure anything calling this function is updated to handle it.